carvertical-hw/src/domains/services/private.http.service.ts

```diff
@@ -0,0 +1,26 @@
+import createLogger from 'pino';
+
+import { invokeLambda } from '../../core/aws/lambda/commands';
+import { config } from '../../core/config';
+
+const logger = createLogger();
+
+const lambdaName = config.aws.lambdas.streamerName;
+
+export async function startStream(
+	keywords: string[],
+): Promise<void> {
+	logger.info('Signal to start the stream');
+
+	const result = await invokeLambda({ keywords }, lambdaName || '');
+
+	if (result.StatusCode !== 202) {
+		logger.error(result, 'Signal not sent');
```

Comment:
```suggestion
		logger.error({ result }, 'Signal not sent');
```

if you want to add some other properties this will be easier.

btw, extract the actual error from the result and do something like { result, error }